src/openrct2/CmdlineSprite.cpp

```diff
@@ -292,15 +293,15 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
         else if (argc == 2)
         {
             const char* spriteFilePath = argv[1];
-
-            if (!sprite_file_open(spriteFilePath))
+            auto sFile = sprite_file_open(spriteFilePath);
+            if (!sFile.has_value())
```

Comment:
(This is on commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/030ba011fa142136fd7ba7a9538911c9679a5eec)

FYI: In the rest of the code we generally compare to nullopt, I think

Reply:
pretty sure we just do `if (!sFile)`. I've started to think that has_value is more clear though

Reply:
> pretty sure we just do if (!sFile)

I don't think so, because I prefer that over the comparison to nullopt or nullptr, but I was dissuaded at the time :D

But `has_value()` is clearer, sure.